scripts/pages/leaderboards/leaderboards.ts

```diff
@@ -272,6 +273,14 @@ class LeaderboardsHandler {
 		this.initTracksDropdown();
 	}
 
+	leaderboardTrackIdSort(a: Leaderboard, b: Leaderboard): number {
+		if (a.trackType === b.trackType) {
+			return a.trackNum - b.trackNum;
+		}
+
+		return a.trackType - b.trackType;
+	}
```

Comment:
If you're only using this in one place I think it's *much* cleaner to inline a sort fn like this. But likely we're use this again in the future anyway - iirc I wrote something like this recently, though don't remember if in website or panorama code. I'd pull it out to `common/leaderboard` and name `sortTrackId`.